lib/tasks/manageiq/ui_tasks.rake

```diff
@@ -7,9 +7,14 @@ namespace :update do
           next
         end
         puts "== #{engine.name} =="
+        require 'json'
+        package_json = JSON.load_file("./package.json")
+        package_json["resolutions"] ||= {}
+        package_json["resolutions"]["es-abstract"] = "~1.21.3"
+        File.write("./package.json", JSON.pretty_generate(package_json))
```

Comment:
ah crap I merged and forgot this was still in here - I shouldn't be merging after a long day at 5:30 😆

Reply:
Did the tests pass? 😆